app/src/main/res/values/styles.xml

```diff
@@ -36,5 +36,8 @@
         <item name="android:fitsSystemWindows">true</item>
     </style>
 
-
+    <style name="ProgressBar" parent="android:Widget.ProgressBar.Large">
```

Comment:
We should use Widget.AppCompat.ProgressBar for showing modern progress bar ♻️

Reply:
🙇